pytest_pootle/fixtures/views.py

```diff
@@ -134,21 +134,18 @@ def project_views(request, client):
     return test_type, project, response.wsgi_request, response, kwargs
 
 
-@pytest.fixture(params=TP_VIEW_TESTS.keys())
-def tp_view_test_names(request):
-    return request.param
-
-
 @pytest.fixture(params=(None, 'member', 'admin'))
-def tp_view_usernames(request):
+def username(request):
```

Comment:
hi @julen i also need parametrized site_users in tests and have added a commit in a PR for this - https://github.com/translate/pootle/pull/4483. Perhaps this could be useful for this test too